<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_note</class>
 <widget class="QWidget" name="add_note">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="note_text_PTE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Текст заметки</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="keys_PTE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ключи поиска</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>16</hour>
     <minute>30</minute>
     <second>0</second>
     <year>2019</year>
     <month>11</month>
     <day>9</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>11</month>
     <day>9</day>
    </date>
   </property>
   <property name="time">
    <time>
     <hour>16</hour>
     <minute>30</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="title_PTE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Заголовок</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
